.gitignore

```
@@ -0,0 +1,2 @@
+**.deb
+orocos_knematics_dynamics
```

```suggestion
orocos_kinematics_dynamics
```
I venture to suggest that there's a typo here :wink: